Doc/library/argparse.rst

```diff
@@ -26,6 +26,67 @@ module also automatically generates help and usage messages and issues errors
 when users give the program invalid arguments.
 
 
+Summary
+-------
+
+Core Functionality
+^^^^^^^^^^^^^^^^^^
+
+The :mod:`argparse` module's support for command-line interfaces is built from the following:
+
+The :class:`argparse.ArgumentParser` creates a new :class:`ArgumentParser` object. Commonly used arguments include `prog`_, `description`_, and `formatter_class`_. For example, the user can create an instance of :class:`ArgumentParser` through the following::
```

Comment:
1. Do we have to write `\`prog\`_` instead of `prog_`?
2. Can you wrap these long lines?

Reply:
Committed the fixes! Wrapped long lines and changed ``` `prog`_ ``` to ``` prog_ ``` for consistency.